app/partials/shapeshift/checkout.pug

```diff
@@ -15,7 +15,7 @@
           p.blue.f-18.em-600(translate="SIMPLE_SEAMLESS_SECURE")
           p.f-16.em-300(translate="{{ vm.trades.length ? 'SHIFT.EXCHANGE_NOTICE' : 'SHIFT.WELCOME_INFO' }}")
       div.width-50.border-desktop.pv-30.no-padding-mobile(ng-if="vm.onStep('confirm')" ng-controller="ShapeShiftConfirmController")
-        shift-confirm(on-complete="onComplete(trade)" on-cancel="onCancel()" payment="vm.payment" fee="vm.fee" quote="vm.quote" handle-shift="shiftHandler(payment)")
+        shift-confirm(on-complete="onComplete(trade)" on-cancel="onCancel()" on-expire="onExpire()" payment="vm.payment" fee="vm.fee" quote="vm.quote" handle-shift="shiftHandler(payment)")
```

Comment:
onExpire() could be shortened to `vm.goTo('create')`. Just a thought

Reply:
Sure, will make the change